apps/api/src/app/inbox/usecases/update-preferences/update-preferences.usecase.ts

```diff
@@ -130,6 +130,7 @@ export class UpdatePreferences {
           name: workflow.name,
           critical: workflow.critical,
           tags: workflow.tags,
+          data: workflow.data,
```

Comment:
the `tags` and `data` fields should be returned from the workflow preferences in v1, as the NC `preferencesFilter` could be used to filter the preferences by this information